index.js

```diff
@@ -32,18 +32,17 @@ class Mapeo extends events.EventEmitter {
     newObs.timestamp = (new Date().toISOString())
     newObs.created_at = (new Date()).toISOString()
 
-    this.osm.create(newObs, function (err, _, node) {
+    this.osm.create(newObs, function (err, node) {
       if (err) return cb(err)
-      newObs.id = node.value.k
-      newObs.version = node.key
-      cb(null, newObs)
+      cb(null, node)
     })
   }
 
   observationGet (id, cb) {
-    this.osm.get(id, function (err, obses) {
+    this.osm.get(id, function (err, elms) {
+      console.log(elms)
       if (err) return cb(err)
-      else return cb(null, flatObs(id, obses).map(transformOldObservation))
+      else return cb(null, elms.map(toObs).map(transformOldObservation))
```

Comment:
the toObs function is to get back version and id from the top-level, because working with the data depends upon these two fields.